src/include/simeng/Core.hh

```diff
@@ -46,7 +85,8 @@ class Core {
   /** Signals core to stop executing the current process.
    * Return Values :
    *  - True  : if succeeded in signaling interrupt
-   *  - False : interrupt not scheduled due to on-going exception or system call
+   *  - False : interrupt not scheduled due to on-going exception or system
+   * call
    */
```

Comment:
Please put `*/` on the last line of the comment which contains the word "call".

Reply:
Re-formatted